config/main.py

```diff
@@ -419,25 +472,7 @@ def _stop_services():
         'hostcfgd',
         'nat'
     ]
-    generated_services_list = _get_sonic_generated_services()
-
-    if generated_services_list is None:
-        log_error("Failed to get generated services")
-        return
-
-    if asic_type == 'mellanox' and 'pmon' in services_to_stop:
-        services_to_stop.remove('pmon')
```

Comment:
i do not see this code in your new code? is it missing?

Reply:
added back, accidentally removed in previous commit.